DB.php

```diff
@@ -1,93 +1,112 @@
 <?php
-	include "config.php";
-	$action = isset($_GET["action"]) ? $_GET["action"] : null;
-	switch ($action) {
-		case 'add':
-			try {
-				$stmt = $conn->prepare("INSERT INTO students (student_code, class_code_id, name, sex, address, date_birth) VALUES (:student_code, :class_code_id, :name, :sex, :address, :date_birth)");
+include 'config.php';
+$action = isset($_GET['action']) ? $_GET['action'] : null;
```

Comment:
```suggestion
$action = $_GET['action'] ?? null;
```
https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op